site/src/nav/DownloadButton.jsx

```diff
@@ -86,10 +88,13 @@ function DownloadButton() {
         onClick={handleDownloadClick}
       >
         <div className="wrapper">
-          <div className="icon">
-            <img className={"dl-img"} src={downloadIcon} />
+          <div className="download-icon">
+             {!loading ? <img className={"dl-img"} src={downloadIcon}/>
+              : <RefreshIcon/>}
           </div>
-          <div>{loading ? "Downloading..." : "Download Visible"}</div>
+          <div className="download-text">
+            {loading ? "Downloading..." : "Download Visible"}
+            </div>
```

Comment:
Can you install prettier extension and set up format on save? The indentation here looks off.

Reply:
Prettier saves a lot of time and makes sure we don't end up with unnecessary diffs.